### PR TITLE
Tango skin :: hidpi fixes

### PR DESCRIPTION
**Pull request by ronso0**
I fixed some major bugs appearing when Tango skin is scaled up (see [lp:1751290](https://bugs.launchpad.net/mixxx/+bug/1751290)), as well as some minor UX issues

- qss: split 'background' shorthand into 'background-color' and 'image' (otherwise those images stayed at 100%)
- Fx flow graphics (were completely gone, width of 'border-image' was not scaled)
- Effect selector (removed explicit 'size', added background to effect list toggle, #1518 also fixes the effects list display)
- Fx parameter row (rebuild, margin is not scaled but was necessary for previous stacked layout)
- beatsize spinboxes, buttons & spinbox in Library
- borders in sampler and preview deck

Screenshot 1920x1080, mixxx & system font at 200%
tango-hidpi-x2_2018-02-27

### CODE REVIEW

**Comment by ronso0** [comment]
Don't merge yet, I need to fix Artist/Time row and Overview.

**Comment by ronso0** [comment]
Ready to test!

**Comment by ronso0** [comment]
I just finished this PR.
Besides the scaling fixes, there are a few minor changes:
- FX unit labels and show_parameters toggle have a background now, so they visually relate more to the actual FX unit. Same applies to sampler row expand toggle
- toggle for additional, hidden deck controls (quantize, repeat etc.) now sits at the outside of deck buttons row
- right-hadn decks' Artist/Title are aligned right (fixes A. of [lp:1750902](https://bugs.launchpad.net/mixxx/+bug/1750902))
- I removed Library toggle from the top row. Didn't use it much, Alt+Tab is much quicker to hide tracks and playlists from snoopy fellows

**Comment by Be-ing** [comment]
LGTM, thanks!

File: res/skins/Tango/fx_toggle_selector.xml [hunk 1]
```diff
@@ -51,10 +51,10 @@ Variables:
       </Template>
 
       <EffectSelector>
-        <Size>82f,20f</Size>
```
Comment (Be-ing) [inline res/skins/Tango/fx_toggle_selector.xml]: This broke the scaling with #1518. I'll revert it in that PR.

**Comment by ronso0** [comment]
it seems this was merged too early: I just discovered a bug with some button icons after I fixed those to scale well. All pause icons appear blurred, although at least their height matches the buttons' height.

Fixed in #1531

**Pull request merged**